fourmolu.yaml

```
@@ -0,0 +1,13 @@
+indentation: 2
+function-arrows: leading
+comma-style: leading
+import-export-style: diff-friendly
+indent-wheres: true
+record-brace-space: true
+newlines-between-decls: 1
+haddock-style: single-line
+let-style: auto
+in-style: right-align
+respectful: true
+fixities: []
+unicode: never
```

```suggestion
function-arrows: trailing
indent-wheres: true
record-brace-space: true
```

We can remove the default values and only leave those that are changed. Also, let's try `function-arrows: trailing`, I think it's more methodical.